src/test/java/de/tum/in/www1/artemis/AbstractSpringIntegrationLocalCILocalVCTest.java

```diff
@@ -174,7 +173,7 @@ public void mockUpdatePlanRepositoryForParticipation(ProgrammingExercise exercis
     }
 
     @Override
-    public void mockUpdatePlanRepository(ProgrammingExercise exercise, String planName, String repoNameInCI, String repoNameInVcs, List<String> triggeredBy) {
+    public void mockUpdatePlanRepository(ProgrammingExercise exercise, String planName, String repoNameInCI, String repoNameInVcs) {
```

Comment:
Suggestion: throw an `UnsupportedOperationException` or something similar. Having an empty method here seems odd.

Reply:
It's not an exceptional behaviour. We just skip it.